.travis.yml

```diff
@@ -27,4 +27,4 @@ deploy:
     tags: true
     condition: "$TOXENV = quality"
   password:
-    secure: FAkEpASSWoRDhERETobEREplACED
+    secure: Hty2+baIiGDmR7eLkwkKI2oweCKEpGG1ULdwUXJGlnrNnowlwHNhEMTmqK0DylQk12w1P8DwOht7LbAb66pvBSqs+nVOnvgwrHc/ELklD1xsPgAj9YIj2FMWW3PdSBHmyGjStJGmnpHK31fDUGeLVXPs5clHwvFt/DB8cwP6Q3/pUDsuxBAmLLFyELR9fTK37gfvrSfaHhYTCYwhA0ZKFs1E8lO6nIk3ZoPOMv5mW1ivjHvN8/qZ7xMTTBhUMTXD6X2iF2r1SluiZfzcOGbb09QLg1x8YBIfgziF+C9ryO1wenzT/VNRPBxu/eF+JYLfx3npJgaIoSnWMR/hSIx4DbFPx7lr/qp+c3WR38g/xEd8wGAYPbXbEcEQRaD4pZpkizoDW5TED10KipsigK27+26JPpQODfW+chWSCi0iFyduX35cbO0Hnt1m/Dmap4MpjhET5BUAdiyinaI8Hxt49TQLzumaY9r6QFWG/LhzS+opGAXCw0DdKvLAmc0zB/S9IW3oD2NIWk3uHPi/bNwp+XX5NGrYlvz+v9cfUCJXnYKIY4K9mtFnBpg/DdC5tfpMOOC5OePwUCtxPMeXEPRC9tgONBRqEX23AO3JkJ0XSyb42hfC/gLKtM5KWo99mMsSLZxpES5pw30bC/UtE8bfgM9STrvUzbokDu4dgj5D3uw=
```

Comment:
Can you add a comment indicating what this secure value is for?  Since you can have multiple of these, it would be useful to know what the purpose of this secret is.  eg.  Creds for Travis/Greenkeeper/etc.

Reply:
we've moved this file to a separate pr, where we've addressed the comment, let us know if that's sufficient: https://github.com/edx/edx-rbac/pull/2/files